File: app/src/main/java/com/pixplicity/cryptogram/activities/SettingsActivity.java
package com.pixplicity.cryptogram.activities;

import android.app.Activity;
import android.content.Context;
import android.content.Intent;
import android.os.Bundle;
import android.support.annotation.NonNull;

import com.crashlytics.android.answers.Answers;
import com.crashlytics.android.answers.ContentViewEvent;
import com.pixplicity.cryptogram.CryptogramApp;
import com.pixplicity.cryptogram.R;

public class SettingsActivity extends BaseActivity {

    public static Intent create(Context context) {
        return new Intent(context, SettingsActivity.class);
    }

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_settings);

        mToolbar.setTitle(R.string.settings);

        Answers.getInstance().logContentView(new ContentViewEvent().putContentName(CryptogramApp.CONTENT_SETTINGS));
    }

    @NonNull
    @Override
    protected Class<? extends Activity> getHierarchicalParent() {
        return CryptogramActivity.class;
    }

}

File: app/src/main/java/com/pixplicity/cryptogram/activities/SettingsActivity.kt
package com.pixplicity.cryptogram.activities

import android.content.Context
import android.content.Intent
import android.os.Bundle

import com.crashlytics.android.answers.Answers
import com.crashlytics.android.answers.ContentViewEvent
import com.pixplicity.cryptogram.CryptogramApp
import com.pixplicity.cryptogram.R

class SettingsActivity : BaseActivity() {

    companion object {
        fun create(context: Context): Intent {
            return Intent(context, SettingsActivity::class.java)
        }
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_settings)

        mToolbar.setTitle(R.string.settings)

        setHomeButtonEnabled(true)

        CryptogramApp.instance.firebaseAnalytics.setCurrentScreen(this, CryptogramApp.CONTENT_SETTINGS, null)
        Answers.getInstance().logContentView(ContentViewEvent().putContentName(CryptogramApp.CONTENT_SETTINGS))
    }

}
